core/core.mk

```diff
@@ -29,12 +29,19 @@ cppflags$(sm)	+= -Ilib/libutee/include
 # configuration file
 
 conf-file := $(out-dir)/core/include/generated/conf.h
+conf-in-file := $(out-dir)/conf.in
```

Comment:
I suggest `$(out-dir)/core/conf.in` because this is the configuration for the TEE core only.